File: 240129/숫자 차례로 출력하기/to-output-numerically.java
import java.util.Scanner;

public class Main {
    static void printnum(int n, int i) {
        if(n == 0) {
            System.out.println();
            return;
        }
        System.out.print(i + " ");
        printnum(n-1, i + 1);
        System.out.print(i +" ");
    }
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int i = 1;
        int n = sc.nextInt();
        printnum(n, i);
    }
}

File: 240129/숫자 차례로 출력하기/to-output-numerically.py
def printnum(n, i):
    i += 1
    if n == 0:
        print()
        return
    print(i, end = " ")    
    printnum(n - 1, i)
    print(i, end = " ")

i = 0
k = int(input())
printnum(k, i)